python/seldon_core/batch_processor.py

```diff
@@ -500,6 +655,14 @@ def _send_batch_feedback(
     type=str,
     help="Unique batch ID to identify all datapoints processed in this batch, if not provided is auto generated",
 )
+@click.option(
+    "--batch-size",
+    "-u",
+    envvar="SELDON_BATCH_SIZE",
+    default=int(1),
```

Comment:
Just `default=1` would suffice, no need to cast onto `int` in Python.
Could be also that it defaults to `None` and that just means no micro-batching.